src/main.rs

```diff
@@ -315,7 +315,7 @@ enum Command {
     /// Command has been deprecated, use `cargo contract build` instead
     #[structopt(name = "generate-metadata")]
     GenerateMetadata {},
-    /// Check that the code builds as Wasm; does not output any build artifact to the top level `target/` directory
+    /// Check that the code builds as Wasm; does not output any `<name>.contract` artifact to the `target/` directory
```

Comment:
We invoke `cargo check` with `--target-dir`. This actually results in a `target` folder in the `target-dir`.
```
--target-dir directory
Directory for all generated artifacts and intermediate files.
```

You can verify it locally:
```
$ cargo new foo
     Created binary (application) `foo` package
$ cd foo
$ cargo check
ls
    Checking foo v0.1.0 (/tmp/foo)
    Finished dev [unoptimized + debuginfo] target(s) in 2.10s
$ ls
Cargo.lock  Cargo.toml  src  target
```